am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/FilterResourceIntegrationTest.java

```diff
@@ -103,4 +107,81 @@ void whenNoExplicitAccessAndRoleHasExplicitAccessTypeShouldReturnNull() {
 
         assertThat(result).isNull();
     }
+
+    @Test
+    void whenListOfResourcesShouldReturnListFilterResourceResponse() {
+        rolesService.addRole(ROLE_NAME, RoleType.RESOURCE, SecurityClassification.PUBLIC, AccessType.ROLE_BASED);
+        rolesService.grantDefaultPermission(createDefaultPermissionGrant(ROOT_ATTRIBUTE, READ_PERMISSION));
+
+        List<Resource> resources = ImmutableList.<Resource>builder()
+            .add(createResource(resourceId))
+            .add(createResource(resourceId))
+            .add(createResource(resourceId))
```

Comment:
I think having list of 2 is good enough. Adding 3rd one does not change anything just makes tests longer to read.